src/components/YourCompany/NameTranslations/AddNameTranslation.vue

```diff
@@ -21,15 +21,24 @@
       <v-row>
         <v-col>
           <div class="action-btns">
-            <v-btn large color="primary"
+            <v-btn v-if="!isAddingTranslation" large outlined color="error"
+                id="name-translation-remove"
+                 @click="emitRemoveName(nameIndex)"
+              >
```

Comment:
Spacing is a little off here, probably was like this before you got here so no biggie. 
Usually if we had multi-line component properties we would align them under the initial property.
ie `<v-btn v-if="!isAddingTranslation"
                  large outlined color="error"
                  id="name-translation-remove"
                  @click="emitRemoveName(nameIndex)"
      >`
      
      or something along those lines :)